bqskit/compiler/passes/partitioning/scan.py

```diff
@@ -135,38 +220,38 @@ def run(self, circuit: Circuit, data: dict[str, Any]) -> None:
             for q in range(circuit.get_size())
         ]
 
+        # Create the block_list, member_dict, and score_heap
+        # `block_list` maintains the current region and score fore each
+        # `qudit_group`.
+        block_list: list[Block]
+        block_list = []
+        # `member_dict` has key: an qudit in the Circuit and value: a list of
```

Comment:
The documentation here could be a little clearer. For example, one way of rewording it may be: `member_dict maps XXX to XXX such that XXX`.

Reply:
Reworded this.